src/utils/datetime/validator.ts

```diff
@@ -0,0 +1,42 @@
+import {isMatch} from 'date-fns'
+import {DEFAULT_TIME_FORMAT} from 'src/shared/constants'
+
+const formatterToDateFnsMap = {
+  'YYYY-MM-DD HH:mm:ss': 'yyyy-MM-dd HH:mm:ss', // DEFAULT_TIME_FORMAT
```

Comment:
I tried replacing the `'YYYY-MM-DD HH:mm:ss'` with `DEFAULT_TIME_FORMAT` but it would not let me access it with `formatterToDateFnsMap['YYYY-MM-DD HH:mm:ss']. Perhaps a JS thing?